src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -14,6 +28,30 @@
 
     @Override
     public int getSize() {
-        return 0;
+        return size;
+    }
+
+    private int getHash(K key) {
+        if (key == null) {
+            return 0;
+        }
+        return Math.abs(key.hashCode() % table.length);
+    }
+
+    private void resize() {
+        Node<K, V>[] currentTable = table;
+        table = new Node[currentTable.length * RESIZE_NUMBER];
+    }
+
+    private static class Node<K, V> {
+        final K key;
+        final V value;
+        final Node<K, V> next;
+
+        public Node(K key, V value, Node<K, V> next) {
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
     }
```

Comment:
The `Node` class is defined correctly. It's parameterized and has final fields `key`, `value` and `next`. But it's not used anywhere. You should use it in your `put` and `getValue` methods.